### PR TITLE
UXG-1253: Add listing card

### PR DESCRIPTION
**Pull request by sgreenl5**
UXG-1253, added the listing card component

### CODE REVIEW

File: src/vue/backend/src/templates/listingsCard.vue [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<template>
+    <checkbox-label
+        :option="option"
+        @checked="$emit('listingSelected', [$event.data, property.listingID])"
```
Comment (ErikIDX) [inline src/vue/backend/src/templates/listingsCard.vue:4]: Events should be kebab-case per Vue docs: https://vuejs.org/v2/guide/components-custom-events.html#Event-Names

File: src/vue/backend/src/templates/listingsCard.vue [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            <idx-single-property-card
+                customClass="import-listing__card"
+                :listingPrice="property.listingPrice"
+                :bedrooms="property.bedrooms"
+                :totalBaths="property.totalBaths"
+                :sqFt="property.sqFt"
+                :address="property.address"
+                :cityName="property.cityName"
+                :courtesy="property.courtesy"
+                :image="property.image[1]"
```
Comment (ErikIDX) [inline src/vue/backend/src/templates/listingsCard.vue:17]: If we don't have guards to check if `property.image` and `property.image[1]` exist for lines 17 and 18, then we'll get runtime errors when it doesn't

**Review by ErikIDX — CHANGES_REQUESTED** [review]

**Review by ErikIDX — APPROVED** [review]

File: src/vue/backend/src/templates/listingsCard.vue [hunk 3]
```diff
@@ -0,0 +1,64 @@
+<template>
+    <checkbox-label
+        :option="option"
+        @checked="$emit('listing-selected', [$event.data, property.listingID])"
+    >
+        <template v-slot:content>
+            <!-- Prop bindings will need to be reevaluated once we have the property shape -->
+            <idx-single-property-card
+                customClass="import-listing__card"
+                :listingPrice="property.listingPrice"
```
Comment (ralls) [inline src/vue/backend/src/templates/listingsCard.vue:10]: You could probably simplify this a bit:
```
<idx-single-property-card
  customClass="import-listing__card"
  v-bind="property" // uses object spread to distribute props to component
/>
```

File: src/vue/backend/src/templates/listingsCard.vue [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+                :bedrooms="property.bedrooms"
+                :totalBaths="property.totalBaths"
+                :sqFt="property.sqFt"
+                :address="property.address"
+                :cityName="property.cityName"
+                :courtesy="property.courtesy"
+                :image="property.image"
+                :primaryPhoto="property.image"
```
Comment (ralls) [inline src/vue/backend/src/templates/listingsCard.vue:18]: I don't see `primaryPhoto` as a prop in `SinglePropertyCard.vue`

**Review by ralls — CHANGES_REQUESTED** [review]
nearly there. nice work.

File: src/vue/backend/src/templates/listingsCard.vue (hunk 1 above)
Comment (ralls) [reply src/vue/backend/src/templates/listingsCard.vue]: Where does it say event names should be kebab-case? It simply states that the emitted event name should match the listener identically, noting that listeners are auto transformed to lowercase due to case insensitivity.

**Pull request merged**